spec.html

```diff
@@ -6961,9 +6957,8 @@ <h1>SetMutableBinding ( _N_, _V_, _S_ )</h1>
 
         <emu-clause id="sec-global-environment-records-getbindingvalue-n-s">
           <h1>GetBindingValue ( _N_, _S_ )</h1>
-          <p>The concrete Environment Record method GetBindingValue for global Environment Records returns the value of its bound identifier whose name is the value of the argument _N_. If the binding is an uninitialized binding throw a *ReferenceError* exception. A property named _N_ normally already exists but if it does not or is not currently writable, error handling is determined by the value of the Boolean argument _S_.</p>
+          <p>The GetBindingValue concrete method of a global Environment Record _envRec_ takes arguments _N_ (a String) and _S_ (a Boolean). It returns the value of its bound identifier whose name is the value of the argument _N_. If the binding is an uninitialized binding throw a *ReferenceError* exception. A property named _N_ normally already exists but if it does not or is not currently writable, error handling is determined by _S_. It performs the following steps when called:</p>
```

Comment:
```suggestion
          <p>The GetBindingValue concrete method of a global Environment Record _envRec_ takes arguments _N_ (a String) and _S_ (a Boolean). It returns the value of the bound identifier whose name is given by _N_. When _S_ is *true*, GetBindingValue may throw. It performs the following steps when called:</p>
```

Reply:
I disagree with the `its` -> `the` change. There may be many such bindings in existence.

Also, this prose was present originally. I don't think this PR needs to do the work of rewording it.

Reply:
This PR should leave the preambles in a state we're happy with, not a state where we're aware of improvements we want to make to them.

I don't want to use "its" because "it" is already used in the sentence to refer to the method. We could use `_envRec_'s` if you don't like "the", but I think "the" is plenty clear enough when read in the context of this preamble.

Reply:
The point of this PR is to standardize preambles as set out in issue #1981. From that standpoint, @bakkot is right that rewording of the non-standardized part of the preambles is out-of-scope. That said, I don't mind making such changes here if we can agree on them.

Re `its` -> `the`: I agree that `its` is odd there. I'm mostly fine with `the`, and `_envRec_'s` seems okay too.

Re the rest, ditto most of my comments under `SetMutableBinding` above.